novendor.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eu
+# Bash replacement for glide novendor command
+
+DIRS=$(ls -ld */ . | awk {'print $9'} | grep -v vendor)
+
+for DIR in ${DIRS}; do
+    GOFILES=$(find ${DIR} -name "*.go")
```

Comment:
Use `git ls-files`? Its behaviour will not vary cross-platform.

See also https://github.com/aelsabbahy/goss/pull/403/files#diff-b67911656ef5d18c4ae36cb6741b7965R7

Reply:
Implemented it, thank you for your feedback!